examples/all-clusters-app/all-clusters-common/all-clusters-app.matter

```diff
@@ -3084,7 +3084,7 @@ server cluster TestCluster = 1295 {
   }
```

Comment:
DO NOT REVIEW!

I did the wrong things I will push update after regen is complete